pkg/util/constants/constants.go

```diff
@@ -20,16 +20,13 @@ const (
 	// MysqlPort is the default mysql port.
 	MysqlPort = 3306
 
-	// HelperXtrabackupPort is the port on which we serve backups
-	HelperXtrabackupPort = 3307
-
 	// OrcTopologyDir path where orc conf secret is mounted
 	OrcTopologyDir = "/var/run/orc-topology"
 
-	// HelperServerPort represents the port on which http server will run
-	HelperServerPort = 8088
-	// HelperServerProbePath the probe path
-	HelperServerProbePath = "/health"
+	// SidecarServerPort represents the port on which http server will run
+	SidecarServerPort = 8007
```

Comment:
Let's use a more common HTTP port.

```suggestion
	SidecarServerPort = 8080
```